apps/client/src/components/Location/Location.js

```diff
@@ -111,15 +112,12 @@ const Location = ({ topic, finishedLocation, setFinishedLocation }) => {
           informatie gevonden:
         </Paragraph>
 
-        <RegisterLookupSummary
-          displayZoningPlans={!useSTTR}
-          address={address}
-        />
+        <RegisterLookupSummary displayZoningPlans={useOLO} address={address} />
 
         <Paragraph>
-          {useSTTR
-            ? `We gebruiken deze informatie bij het invullen van de vergunningcheck. `
-            : `U hebt deze informatie nodig om de vergunningcheck te doen op het Omgevingsloket. `}
+          {useOLO
+            ? `U hebt deze informatie nodig om de vergunningcheck te doen op het Omgevingsloket. `
+            : `We gebruiken deze informatie bij het invullen van de vergunningcheck. `}
```

Comment:
I already changed the texts and layout in my PR #335 so can you undo these change to prevent conflicts?

Reply:
will do.